server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -1029,18 +1031,21 @@ private static class LogMeta {
         private final int replicaId;
         private final long timestamp;
         private final int size;
+        private final boolean compressed;
         private final List<Long> blocks = new ArrayList<>();
 
         @JsonCreator
         LogMeta(@JsonProperty(value = "replicaId", required = true) int replicaId,
                 @JsonProperty(value = "timestamp", defaultValue = "0") Long timestamp,
-                @JsonProperty("size") int size) {
+                @JsonProperty("size") int size, @JsonProperty("compressed") Boolean compressed) {
```

Comment:
`@Nullable`?
Shouldn't we create a PR for forward compatibility first?

Reply:
Fixed.